scipy/stats/_continuous_distns.py

```diff
@@ -6123,6 +6125,43 @@ def _stats(self, b, moments='mv'):
     def _entropy(self, c):
         return 1 + 1.0/c - np.log(c)
 
+    def fit(self, data, *args, **kwds):
+        data = np.asarray(data)
```

Comment:
This is done in _check_fit_input_parameters. Maybe that should return `data` so you don't do it twice?